app/assets/images/icons.svg

```diff
@@ -0,0 +1,242 @@
+<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
are the icons something that we could import from the UI-components lib instead of copying the file over? Might be easier to keep it up-to-date that way? and visualize what icons we have access to?